.drone.yml

```diff
@@ -15,6 +15,27 @@ steps:
   - name: docker
     path: /var/run/docker.sock
 
+- name: fossa
+  image: registry.suse.com/suse/sle15:15.3.17.8.1
```

Comment:
We've decided to just do `15:15.3` instead of the full version. I know for this case it doesn't really matter, but some of these isolated tags in the past were buggy.